guides/common/modules/proc_invalidating-jwts-of-other-users.adoc

```diff
@@ -0,0 +1,36 @@
+[id="invalidating-jwts-of-other-users"]
+= Invalidating JWTs of other users
+
+You can invalidate all registration JSON Web Tokens of one or more users.
```

Comment:
can you also mention registration tokens as an alternative term for JWT tokens.

Reply:
Yes, it's in the chapter introduction.